torch/_inductor/codegen/wrapper.py

```diff
@@ -666,9 +671,23 @@ def generate_user_defined_triton_kernel(
         stream_name = self.write_get_raw_stream(
             V.graph.scheduler.current_device.index, V.graph
         )
-        self.writeline(
-            f"{kernel_name}.run({', '.join(args)}, grid={grid}, stream={stream_name})"
-        )
+
+        if profiler._capture_generated_kernel:
+            self.writeline("if autograd_profiler._is_profiler_enabled:")
+            self.writeline(
+                f'    handle = _record_function_with_args_enter("{kernel_name}", getattr({kernel_name}, "__file__", '
+                "),"
```

Comment:
@davidberard98 did you add something along the lines of fast record ? should we add that here or am i miss remembering ?

Reply:
yeah - RecordFunctionFast, but it doesn't record info on the tensors